src/main/java/com/watch/shop/app/model/service/WatchServiceImpl.java

```diff
@@ -0,0 +1,57 @@
+package com.watch.shop.app.model.service;
+
+import com.watch.shop.app.model.entity.Watch;
+import com.watch.shop.app.model.repository.WatchStore;
+
+import java.math.BigDecimal;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+import static java.util.Objects.isNull;
+
+public class WatchServiceImpl implements WatchService {
+
+    public static final String PRICE_PARAMETER = "price";
+    public static final String COLOR_PARAMETER = "color";
+    public static final String ARRIVAL_DATE_PARAMETER = "arrival date";
+
+    private final WatchStore watchStore = new WatchStore();
+
+    public List<Watch> getWatches() {
+        return Collections.unmodifiableList(watchStore.getWatches());
+    }
+
+    public List<Watch> sortWatchesByParam(String param) {
+        if (isNull(param) || param.isBlank()) {
+            throw new IllegalArgumentException("Param cannot be empty");
+        }
+
+        Comparator<Watch> comparator = resolveComparator(param);
+
+        List<Watch> watches = watchStore.getWatches();
+        watches.sort(comparator);
+
+        return watches;
+    }
+
+    public BigDecimal getTotalCost() {
+        return watchStore.getWatches().stream()
+                .map(Watch::getPrice)
+                .reduce(BigDecimal.ZERO, BigDecimal::add);
+    }
+
+    public void insertWatch(Watch watch) {
+        watchStore.addWatch(watch);
+    }
+
+    private Comparator<Watch> resolveComparator(String param) {
+
```

Comment:
Sometimes (in minority of cases) it could be possible to insert empty line AFTER { opening brace.

But please keep in mind symmetry / consistency principle.

We handle in SAME way, use SAME approach in SAME situation.

1. 
    public BigDecimal getTotalCost() {
        return watchStore.getWatches().stream()
                .map(Watch::getPrice)
                
                no empty line between { and return
                
2
 private Comparator<Watch> resolveComparator(String param) {
 
  return switch (param) {
  
  
 There is empty line.
 
 Same situation.
 Different approaches.
 
 It's rise questions and code became not good to read.
 
 Action point:
 Just remove 49 line (empty line)